backend/src/main/java/com/htecgroup/skynest/controller/UserController.java

```diff
@@ -74,4 +74,12 @@ public List<UserResponse> getUsers() {
         .map(e -> modelMapper.map(e, UserResponse.class))
         .collect(Collectors.toList());
   }
+
+  @DeleteMapping("/delete")
+  public ResponseEntity<String> deleteUser(@RequestParam String uuid) {
```

Comment:
If this is UUID already, than change the type to it, instead of mapping on service layer

Reply:
I had some issues doing it this way, but now it's working just fine. I changed it

Reply:
Maybe is good that uuid is actually path param instead of query param